requirements-test.txt

```diff
@@ -0,0 +1,7 @@
+pytest~=7.3.1
+pytest-asyncio~=0.21.0
+pytest-cov~=4.0.0
+pytest-trio~=0.8.0
+pytest-twisted~=1.14.0
+anyio~=3.6.2
+twisted~=22.10.0
```

Comment:
When running tests locally and on Github Actions, if you don't install these three dependencies, most of the test cases in the unit tests will be skipped (you can also try to see if you encounter a similar situation). This will result in the unit tests not being executed properly. According to the warning message provided by pytest, I later added these three dependencies, and then the unit tests were able to run successfully.